airbyte-config/models/src/main/java/io/airbyte/config/helpers/S3Logs.java

```diff
@@ -52,10 +53,10 @@
   private static S3Client S3;
 
   private static void checkValidCredentials(Configs configs) {
```

Comment:
I would call it `assertValidCredentials` since call this function acts as an assert and you lose control on the execution flow.

Reply:
Done.